index.php

```diff
@@ -225,6 +225,11 @@ function gtag(){dataLayer.push(arguments);}
 <td>Henna</td>
 <td>#2020</td>
 </tr>
+<tr>
+<td>Marissa</td>
+<td>MONDAY, FEBRUARY 10TH 2020</td>
```

Comment:
<img src="https://media.giphy.com/media/13V60VgE2ED7oc/giphy-downsized.gif" />

It looks like you're trying to write a date. Would you like some help?

```suggestion
<td>January Trumpteeth, Twentymemeteen</td>
```

Reply:
what is this

Reply:
That's the date